recipes/cjdk/meta.yaml

```diff
@@ -10,6 +10,11 @@ source:
   sha256: 6c5c9d6df20c2d6f8dee547012b6daeee993e392e44c439cad2850114fcff778
 
 build:
+  # While cjdk itself is pure Python, because it has a `[project.scripts]` entry point
```

Comment:
Ah, if it is just an entry-point, you can fix it by specifying it in the recipe.